scripts/badger/convex_frax_tcl_strategy.py

```diff
@@ -0,0 +1,55 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+# flag
+prod = False
+
+
+# slippage
+COEF = 0.98
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_cow(prod)
+
+    # tokens involved
+    dai = vault.contract(r.treasury_tokens.DAI)
+    fxs = vault.contract(r.treasury_tokens.FXS)
+    crv = vault.contract(r.treasury_tokens.CRV)
+    cvx = vault.contract(r.treasury_tokens.CVX)
+
+    # contracts
+    private_vault = vault.contract(r.convex.frax.private_vaults.badger_fraxbp)
+
+    # snap
+    tokens = [fxs, crv, cvx]
+    vault.take_snapshot(tokens)
+
+    # NOTE: vault contains crv & cvx > 0, to make sure it is only the strat rewards check balances before
+    crv_balance_before = crv.balanceOf(vault)
+    cvx_balance_before = cvx.balanceOf(vault)
```

Comment:
while technically correct, i would not exclude these tokens and just sell all of it.

looking at the origin, it actually originates from the badgerfraxbp being staked on the convex gauge, from before there was a frax gauge available. so fits well in the purpose of this script imo.

it is not considerable, it cleans up the vault and i would not know what to do with it otherwise.

Reply:
selling whole balance change included in 67307fa33055e1e8249c124ba8645de85061148c